### DISCUSSION TITLE
d with two ascenders

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#133] [General]
Just today I discovered https://collation.folger.edu/2021/09/brevigraphs/. In section 14 you will find examples of "d with two ascenders" (more examples in section 29, one seems identical to Junicode U+F0005, cf. https://groups.google.com/d/msgid/mufi-fonts). I've seen them before quite often in some low quality Polish books and misinterpreted (?) it as an insular d with a dot above. What is the right way to encode it in Unicode/Junicode (possibly with some tags)?

**Comment by psb1558** [comment]
I'm pretty sure that MUFI U+F159 is supposed to stand for this, though the MUFI reference glyph doesn't look a whole lot like the one on the Folger page. If you've got some images handy of this character in manuscripts or early printed books, could you share them here?

**Reply by jsbien** [reply]
I guess you are right. As for examples, cf. fig 21 on p. 9 in http://www.cobrpp.com.pl/actapoligraphica/uploads/pdf/AP2021_Bien.pdf and fig. 33 on p. 33 in http://kwartjez.amu.edu.pl/teksty/teksty2021_1_30/Bien.pdf. I will prepare more images soon.

**Reply by jsbien** [reply]
![Ungler1_pismo01_tab112_d](https://user-images.githubusercontent.com/8024702/165097845-3fb79b6e-25e1-4481-bceb-b790f50e0725.png)
![Ungler1_pismo02_tab113_d](https://user-images.githubusercontent.com/8024702/165097846-c85bd877-448f-4cf2-82f5-4c970a6f556a.png)
![Ungler1_pismo03_tab114_d](https://user-images.githubusercontent.com/8024702/165097848-3c92f040-b7cd-4aaf-8bea-c258590ae699.png)
![Ungler1_pismo04_tab115_d](https://user-images.githubusercontent.com/8024702/165097851-373f59f6-8614-42af-8c75-2cb0c1257499.png)
![Ungler1_pismo05_tab116_d](https://user-images.githubusercontent.com/8024702/165097853-7bab215f-b3e3-4d97-9e09-3872ad851104.png)
![Ungler1_pismo06_tab117_d](https://user-images.githubusercontent.com/8024702/165097854-50d56d7f-ffb2-4c8b-a94a-422689c59985.png)
![Ungler1_pismo07_tab118_d](https://user-images.githubusercontent.com/8024702/165097855-1c22c7da-2fb6-42f5-a13a-14646913eb99.png)
![Ungler2_pismo14_tab357_d](https://user-images.githubusercontent.com/8024702/165097861-4bed6f35-73b1-4b32-b7e4-04a54810d12b.png)
![Ungler2_pismo16_tab359_d](https://user-images.githubusercontent.com/8024702/165097865-553bf79c-3d68-4a3b-b7da-8f2118280a66.png)
![Ungler2_pismo18_tab360_d](https://user-images.githubusercontent.com/8024702/165097866-48cfca43-8e46-461a-bffc-c6b947d83ab5.png)

**Reply by jsbien** [reply]
More examples of the letter d (some with ligatures, sometimes just strange) from two other printers:
![Augezdecki_pismo01b_tab403_d+](https://user-images.githubusercontent.com/8024702/165105013-cc21858b-000f-428d-b522-6709d34fd349.png)
![Augezdecki_pismo01_tab402_d](https://user-images.githubusercontent.com/8024702/165105015-404f59d8-45e1-4435-975d-6c0eb80244a3.png)
![Augezdecki_pismo02a_tab406_d+](https://user-images.githubusercontent.com/8024702/165105017-a35b1be6-343a-43aa-ad5a-e718b8d31429.png)
![Augezdecki_pismo13_tab412_d+](https://user-images.githubusercontent.com/8024702/165105020-343406d4-663d-44ee-92cc-a036e4d93ddb.png)
![Augezdecki_pismo14_tab410_d](https://user-images.githubusercontent.com/8024702/165105021-422460d6-65d3-4a6f-84d8-4eefdd61ce6f.png)
![Augezdecki_pismo15_tab412_d+](https://user-images.githubusercontent.com/8024702/165105023-c4349377-187b-47e2-962a-4d7a6361574a.png)
![Haller_pismo01_tab164_d](https://user-images.githubusercontent.com/8024702/165105026-b1387d5a-25e8-4eec-ae46-bb79f38c0563.png)
![Haller_pismo02_tab165_d](https://user-images.githubusercontent.com/8024702/165105029-b143d149-0591-46f3-96cb-da73a7f14a67.png)
![Haller_pismo03_tab166_d](https://user-images.githubusercontent.com/8024702/165105031-1f819c7a-5245-4d99-83ea-7bfb638ddd36.png)
![Haller_pismo04_tab167_d](https://user-images.githubusercontent.com/8024702/165105032-2257737d-ab75-4587-baf9-efb3f4b255e3.png)
![Haller_pismo05_tab168_d](https://user-images.githubusercontent.com/8024702/165105034-dd78addd-1091-4ae7-a46c-798902c680fc.png)
![Haller_pismo06_tab169_d](https://user-images.githubusercontent.com/8024702/165105035-49f59240-5438-485b-871f-f919a76db814.png)
![Haller_pismo07_tab170_d](https://user-images.githubusercontent.com/8024702/165105036-78f22f63-696a-473a-8409-a42e6f1b287c.png)
![Haller_pismo08_tab171_d](https://user-images.githubusercontent.com/8024702/165105038-0441ef75-8f62-4cea-98a7-bf067136b1ab.png)
![Haller_pismo09_tab172_d](https://user-images.githubusercontent.com/8024702/165105040-d5bfe2c7-4c33-4179-b918-010ccf9f8479.png)
![Haller_pismo10_tab173_d](https://user-images.githubusercontent.com/8024702/165105042-68bb0b5c-d2c0-401b-8ed4-dced3070571d.png)
![Haller_pismo11b_tab174_d](https://user-images.githubusercontent.com/8024702/165105044-a64318d0-3eea-4696-b5cb-956b8acae85f.png)
![Haller_pismo12_tab175_d+](https://user-images.githubusercontent.com/8024702/165105048-ab0c7c04-0650-4189-bd74-74cd7fbcf76a.png)
![Haller_pismo13_tab172_d](https://user-images.githubusercontent.com/8024702/165105049-fc1c6a97-65f7-485f-8bcf-035940e54ba9.png)

**Comment by psb1558** [comment]
Thanks for these. I'm thinking, based on these and entries in Capelli, that one of these shapes would be better for U+F159 than the current one:
<img width="120" alt="image" src="https://user-images.githubusercontent.com/16215767/165148110-3c09adeb-f254-4fed-bc87-a99cfc5ae522.png">
Another possibly related character is MUFI U+EBB2, d rotunda with acute, which may be trying to render the shapes where the flourish stroke does not cross. I wonder if it's better interpreted as d with stroke than d with accent:
<img width="115" alt="image" src="https://user-images.githubusercontent.com/16215767/165148737-b744a758-d050-44dc-bdf6-11e2bc068baa.png">
Opinions?

**Reply by jsbien** [reply]
As for U+F159, I suspect there is some rationale behind the current shape of the glyph, so it looks better to make the additional shapes just the variants. It's a pity MUFI is actually missing the documentation analogical to the Unicode character proposals. Perhaps there is a corpus of MUFI encoded texts for which the scans are easily available and searching in it seems to be the only way to document the reasons for introducing a character.
As for U+EBB2, it looks nice. The terminology doesn't really matter here, but is this a stroke or a comma?
